10.18.288.0 (#300)
SoftwareAG/c8y-releasenotes: content/release-10-18+/devices-10-18+.md

</tr>
</thead><tbody>

<tr>
<td>Device management</td>

> <td>If a pending device in the registration list has been accepted, the device will now be removed from the list - no "accepted" status will be shown.</td>

Reviewer: ```suggestion
<td>If a pending device in the registration list has been accepted, the device is now removed from the list instead of the list showing the "accepted" status for the device.</td>
```
